src/wazuh_modules/syscollector/syscollector_linux.c

```diff
@@ -1059,10 +1059,30 @@ void sys_network_linux(int queue_fd, const char* LOCATION){
             if (strcmp(ifaces_list[i], ifa->ifa_name)){
                 continue;
             }
+
             if (ifa->ifa_flags & IFF_LOOPBACK) {
                 continue;
             }
 
+            /* Get MAC address */
+            char addr_path[PATH_LENGTH] = {'\0'};
+            snprintf(addr_path, PATH_LENGTH, "%s%s/address", WM_SYS_IFDATA_DIR, ifaces_list[i]);
+
+            FILE *fs_if_addr = fopen(addr_path, "r");
+            if (fs_if_addr != NULL) {
+                char MAC[MAC_LENGTH] = {'\0'};
```

Comment:
`char MAC[]` is a variable. According to the naming convention, it should be lower-case.

Reply:
Thanks for spotting this. It has been fixed in the latest commit.